src/main/thrift/parquet.thrift

```diff
@@ -1017,8 +1020,10 @@ struct OffsetIndex {
 }
 
 /**
- * Description for ColumnIndex.
- * Each <array-field>[i] refers to the page at OffsetIndex.page_locations[i]
+ * "Page Index": Optional statistics for each data page in a ColumnChunk.
```

Comment:
I think this is the confusion I have mentioned above. As we can see in https://github.com/apache/parquet-format/blob/master/PageIndex.md, the title is `ColumnIndex xxx` but actually it mentions both ColumnIndex and OffsetIndex as PageIndex.